src/main/java/com/github/manolo8/darkbot/extensions/features/decorators/InstructionProviderDecorator.java

```diff
@@ -39,4 +45,11 @@ public static void showInstructions(Object featureInstance, String name) {
         }
     }
 
+    private boolean isApiCall() {
+        for (StackTraceElement el : new Exception().getStackTrace()) {
+            if (ConfigHandler.class.getName().equals(el.getClassName()) && "changeConfig".equals(el.getMethodName()))
+                return true;
+        }
```

Comment:
This is NOT a valid way to check, and will probably never be true anyways since it happens async. (ie: you set a lazy, and when the lazy gets consumed in the next tick is when the change actually occurs)

Reply:
reverted